visualize_data.ipynb

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
It is easier to compare in the future if the notebooks haven't been run when they are uploaded but we can discuss this later.

Reply:
ooh how do i do this? and is there still a way to show output plots in the github then, which i think are very useful to have uploaded / easily viewable?

Reply:
In order to save without the cells being run, you just need to _Restart & Clear Output_ on the _Kernel_ tab in the notebook. In order to show the output plots, which I agree it is useful, we can simply download the notebook as html and upload that to GitHub. When everything is finalised and it is not going to be changed (Friday), we can simply upload the notebooks with the images. What do you think?

Reply:
sounds good!!